core/components/atoms/button/button.js

```diff
@@ -224,9 +227,11 @@ Button.Element = styled.button`
   pointer-events: ${props => (props.disabled || props.loading || props.success ? 'none' : null)};
   transition: border-color ${misc.animationDuration}, background ${misc.animationDuration};
 
-  ${Icon.Element}, ${StyledSpinner} {
-    position: relative;
-    top: -1px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
I don't think you need to justify this content.

To center the content in the btn, I'd change the padding. The current padding left and right is `0`. I'd make it one of the spacers of the system. 

I'd also remove the `min-width` or leave it ad safety net if you want.

Reply:
PD: We have some cases for the `min-width`

![image](https://user-images.githubusercontent.com/4152942/46165884-1f297c80-c268-11e8-9efc-2b0d028f9021.png)